database/migrations/20190219185900_foreignkey.js

```diff
@@ -0,0 +1,9 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.table('users', table => {
+    table.integer('contacts_id').references('contacts_id').inTable('contacts').notNullable();
+  });
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTable('users');
```

Comment:
You shouldn't be blindly dropping this table. You should be removing the reference in the `contacts_id` column.